pages/itinerary.js

```diff
@@ -67,67 +67,97 @@ export default function Itinerary({ trips, sights, setSights }) {
-                    sight={sight}
-                    sights={sights}
-                    setSights={setSights}
-                    handleDeleteItem={handleDeleteItem}
-                    handleUpdateItem={handleUpdateItem}
-                  />
-                )
-            )}
-            <StyledForm
-              onSubmit={(event) =>
-                handleAddItem(event, format(date, "yyyy-MM-dd"))
-              }
-            >
-              <StyledLabel htmlFor="dateInput">Add Item:</StyledLabel>
-              <StyledInput
-                type="text"
-                id="dateInput"
-                name="itineraryItem"
-                placeholder="Add Item"
-                required
-                pattern="\S+"
-              ></StyledInput>
-              <StyledLabel htmlFor="time">Set time:</StyledLabel>
-              <StyledInput type="time" id="time" name="time"></StyledInput>
-              <StyledButton type="submit">+</StyledButton>
-            </StyledForm>
-          </StyledDay>
-        ))}
-      </StyledItinerary>
-      <StyledLink href="/">Save</StyledLink>
+      <Scrollbox>
+        <Carousel
+          trips={trips}
+          sights={sights}
+          setSights={setSights}
+          handleSortItem={handleSortItem}
+        />
+        <StyledItinerary>
+          {datesArray.map((date) => (
+            <StyledDay key={uid()}>
+              <StyledDate>{format(date, "dd/MM/yy")}</StyledDate>
+              {sortedSights.map(
+                (sight) =>
+                  sight.plannedDate === format(date, "yyyy-MM-dd") && (
+                    <ItineraryItem
+                      key={sight.id}
+                      sight={sight}
+                      sights={sights}
+                      setSights={setSights}
+                      handleDeleteItem={handleDeleteItem}
+                      handleUpdateItem={handleUpdateItem}
+                    />
+                  )
+              )}
+              <StyledForm
+                onSubmit={(event) =>
+                  handleAddItem(event, format(date, "yyyy-MM-dd"))
+                }
+              >
+                <StyledLabel htmlFor="dateInput">
+                  Add Item:
+                  <StyledInput
+                    type="text"
+                    id="dateInput"
+                    name="itineraryItem"
+                    placeholder="Add Item"
+                    required
+                    pattern="\S+"
+                  ></StyledInput>
+                </StyledLabel>
+                <StyledLabel htmlFor="time" className="timeInput">
+                  Set time:
+                  <StyledInput type="time" id="time" name="time"></StyledInput>
+                </StyledLabel>
+                <StyledButton type="submit">+</StyledButton>
+              </StyledForm>
+            </StyledDay>
+          ))}
+        </StyledItinerary>
+      </Scrollbox>
+      <StyledFooter>
+        <StyledLink href="/">Save</StyledLink>
```

Comment:
this actually looks a bit sketchy. The user thinks they "save" something, but in reality they are only redirected to a nother page. Maybe you can change the text and say something like "go to next step" or "finish".